simd.cmake

```diff
@@ -0,0 +1,91 @@
+#
+# Check if SSE4.1 or AVX2 instructions are available on the machine where the
+# project is compiled.
+# These commands are extracted and modified from
+# https://github.com/VcDevel/Vc/blob/master/cmake/FindSSE.cmake
+#
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
```

Comment:
This file should be named `FindSSE.cmake` and be placed into `cmake/modules`.
Not here at the root of the project.

Reply:
Yeah, FindSIMD is good